src/authentication/keychain.ts

```diff
@@ -8,14 +8,8 @@ import { IHostConfiguration } from './configuration';
 function getNodeModule<T>(moduleName: string): T | undefined {
 	const vscodeRequire = eval('require');
 	try {
-		return vscodeRequire(`${vscode.env.appRoot}/node_modules.asar/${moduleName}`);
+		return vscodeRequire('keytar');
```

Comment:
@alexandrudima Are you going to update the shimming, so there is no adoption needed?

Reply:
This change should not be necessary anymore... The shim should work in all cases now...